### PR TITLE
Hello function

### PR DESCRIPTION
**Pull request by MarkDurrheim**

### CODE REVIEW

File: src/bot_controller.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const inter = require('./interface');
+
+module.exports = {
+    handle_call: function (req, res) {
```
Comment (sl1msn1per) [inline src/bot_controller.js]: To the outside world you're exposing bot_controller.handle_call, yet this only really handles saying hello. Perhaps rename this to make it more clear?

File: src/bot_controller.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+        res.send({
+                'text': inter.hi_server(req.body.user_name), //Note: the user_name field is being phased out
+                'response_type': 'in_channel'
```
Comment (sl1msn1per) [inline src/bot_controller.js]: Please add a comment so that I know what 'in_channel' means

File: src/html.js [hunk 3]
```diff
@@ -13,11 +17,7 @@ app.get('/', (req, res) => {
     res.send("Uptime: " + format.duration_str(server_status.uptime()));
 });
 app.post('/gnomeling', (req, res) => {
-    res.send({
-            'text': inter.hello(req.body.user_name), //Note: the user_name field is being phased out
-            'response_type': 'in_channel'
-        }
-    );
+    bot_controller.handle_call(req, res);
 });
```
Comment (sl1msn1per) [inline src/html.js]: As discussed Mark, please reformat the code so as only calls to interface or bot_controller should be made here.

Comment (sl1msn1per) [reply src/html.js]: I'll do mine after this review

**Comment by MarkDurrheim** [comment]
Corrections made. I forgot to write tests though, which I will do now.

**Pull request merged**